demo/tern.html

```
@@ -0,0 +1,95 @@
+
+(function(exports) {
+  exports.randomElt = function(arr) {
+    return arr[Math.floor(arr.length * Math.random())];
+  };
+  exports.strList = "foo".split("");
+  exports.intList = exports.strList.map(function(s) { return s.charCodeAt(0); });
+})(window.myMod = {});
+
+var randomStr = myMod.randomElt(myMod.strList);
+var randomInt = myMod.randomElt(myMod.intList);
+</textarea></p>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </p> ]

(tag-pair)